audit/format.go

```diff
@@ -399,6 +412,10 @@ type AuditRequest struct {
 	Operation                     logical.Operation      `json:"operation,omitempty"`
 	MountType                     string                 `json:"mount_type,omitempty"`
 	MountAccessor                 string                 `json:"mount_accessor,omitempty"`
+	MountRunningVersion           string                 `json:"mount_running_plugin_version,omitempty"`
+	MountRunningSha256            string                 `json:"mount_running_sha256,omitempty"`
+	MountPluginType               string                 `json:"mount_plugin_type,omitempty"`
```

Comment:
Is there any precedent for this name? It's a pretty good name either way. One other name we could consider is `MountClass` which would align with the events system which is another place we report and need to differentiate between both type and class together:

https://github.com/hashicorp/vault/blob/3c2faf269e73330f085a421c3a7739834884a499/sdk/logical/event.pb.go#L-33-L34

Reply:
Resolving, as I think this name is fine as-is.